src/main.rs

```diff
@@ -68,6 +86,23 @@ fn is_relevant_commit(commit: &conventional::Commit) -> bool {
     commit.type_() == "feat" || commit.type_() == "fix"
 }
 
+fn print_version(base: &semver::Version, commits: &Vec<conventional::Commit>) {
+    println!("{}", get_version(base, commits));
+}
+
+fn get_version(base: &semver::Version, commits: &Vec<conventional::Commit>) -> semver::Version {
+    if commits.iter().any(|x| x.breaking()) {
```

Comment:
I'm not 100% that this logic works (since I think prereleases would potentially break it wouldn't it?)

So like:

```
1.0.0
feat
feat
calc version @ 1.1.0
1.1.0-prerelease-doot
feat
feat
fix
fix
calc version (correct = 1.1.0, this = 1.2.0)?
```

Reply:
If you used the prerelease as a base yea. I think you would probably use the last real release as the base for the next real release though ICBC. Honestly the case I’m thinking about today is militant CD, i.e. no prereleases.

Reply:
Fair point just wanted to bring it up